src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsS3Client.cs

```diff
@@ -63,6 +63,12 @@ public RavenAwsS3Client(S3Settings s3Settings, Config.Categories.BackupConfigura
             }
             else
             {
+                if (string.IsNullOrWhiteSpace(s3Settings.AwsRegionName) == false)
+                {
+                    // region for custom server url isn't mandatory
+                    config.RegionEndpoint = RegionEndpoint.GetBySystemName(s3Settings.AwsRegionName);
```

Comment:
Will this work with non-standard region names?

Reply:
Yes. Tested manually. Will add a test.